packages/storefront/src/components/PropsTable.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div>
+    <code v-if="hasBreakpointCustomizableProp" class="code-before-table" v-html="breakpointCustomizableGeneric"></code>
+    <table>
```

Comment:
we could also use `p-table` now

Reply:
Native one gave better result since `p-table` was scrollable with long descriptions.

Reply:
Depending on the circumstances I think it's good when it becomes scrollable, otherwise type definitions might become hyphenated or the table width is larger than the viewport which either zooms the whole page or makes it scrollable horizontally.

Reply:
Default `p-table` will look and behave like this which is a step backwards.

<img width="1258" alt="Bildschirmfoto 2024-01-08 um 13 26 55" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/f91ae053-373e-4fdc-a3d9-a5113ad9ae87">

Having it scrollable doesn't really help smaller viewports, too, unless we add support for fixed 1st column.

<img width="392" alt="Bildschirmfoto 2024-01-08 um 13 33 49" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/32d11c7c-2cca-411c-aa81-5e415f074722">

I think if we want to use `p-table` here, we should use it everywhere for consistency.

Reply:
Ok, but then we'll definitely need to have a look in a follow up issue because the current table implementation is broken for smaller viewports.